apps/app/src/app/redirect/redirect.component.ts

```diff
@@ -1,18 +1,67 @@
 import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { SDKService } from '@flaps/auth';
-import { take } from 'rxjs';
+import { injectScript } from '@flaps/core';
+import { take, tap, concatMapTo } from 'rxjs';
 
+declare var gapi: any;
```

Comment:
var?

Reply:
it is a global variable created by the Google JS client which is injected dynamically so it is not like it is part of our nodes packages, it cannot be resolved by typescript or anything.
If we want typescript to be happy with it, we have to declare it as a global variable and `declare var` is the syntax we are supposed to used (you will also notice the `// <reference path="` things at the beginning which allows to load typings for these dynamically loaded libraries)

Reply:
thanks for the explanations :)